prowler/providers/aws/services/bedrock/bedrock_service.py

```diff
@@ -110,3 +110,58 @@ class Guardrail(BaseModel):
     tags: Optional[list] = []
     sensitive_information_filter: bool = False
     prompt_attack_filter_strength: Optional[str]
+
+
+class BedrockAgent(AWSService):
+    def __init__(self, provider):
+        # Call AWSService's __init__
+        super().__init__("bedrock-agent", provider)
+        self.agents = {}
+        self.__threading_call__(self._list_agents)
+        self.__threading_call__(self._list_tags_for_resource, self.agents.values())
+
+    def _list_agents(self, regional_client):
+        logger.info("Bedrock Agent - Listing Agents...")
+        try:
+            for agent in regional_client.list_agents().get("agentSummaries", []):
+                agent_arn = f"arn:aws:bedrock:{regional_client.region}:{self.audited_account}:agent/{agent['agentId']}"
+                if not self.audit_resources or (
+                    is_resource_filtered(agent_arn, self.audit_resources)
+                ):
+                    self.agents[agent_arn] = Agent(
+                        id=agent["agentId"],
+                        name=agent["agentName"],
+                        arn=agent_arn,
+                        guardrail_id=agent.get("guardrailConfiguration", {}).get(
+                            "guardrailIdentifier"
+                        ),
+                        region=regional_client.region,
+                    )
+        except Exception as error:
+            logger.error(
+                f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+
+    def _list_tags_for_resource(self, agent):
```

Comment:
I think that name of function does not match with the name of the name of his argument. I mean, if the function is called `_list_tag_for_resource` the argument passed should call `resource`. If the argument is called `agent`, then the function should be called `_list_tags_for_agent`

Reply:
Done